tests/unit/transformers/test_utils.py

```diff
@@ -289,54 +314,6 @@ def test_learn_rounding_digits_pyarrow_float():
     assert output == 2
 
 
-def test_learn_rounding_digits_negative_decimals_float():
-    """Test the learn_rounding_digits method with floats multiples of powers of 10.
-
-    If the data has all multiples of 10 the output should be 0.
-
-    Input:
-    - An array that contains floats that are multiples of powers of 10, 100 and 1000 and a NaN.
-    """
-    data = pd.Series(np.array([1230.0, 12300.0, 123000.0, np.nan]))
-
-    output = learn_rounding_digits(data)
-
-    assert output == 0
-
-
-def test_learn_rounding_digits_negative_decimals_integer():
-    """Test the learn_rounding_digits method with integers multiples of powers of 10.
-
-    If the data has all multiples of 10 the output should be 0.
-
-    Input:
-    - An array that contains integers that are multiples of powers of 10, 100 and 1000
-        and a NaN.
-    """
-    data = pd.Series(np.array([1230, 12300, 123000, np.nan]))
-
-    output = learn_rounding_digits(data)
-
-    assert output == 0
-
-
-def test_learn_rounding_digits_all_missing_value_replacements():
-    """Test the learn_rounding_digits method with data that is all NaNs.
-
-    If the data is all NaNs, expect that the output is None.
-
-    Input:
-    - An array of NaN.
-    Output:
-    - None
-    """
-    data = pd.Series(np.array([np.nan, np.nan, np.nan, np.nan]))
-
-    output = learn_rounding_digits(data)
-
-    assert output is None
```

Comment:
Can we keep this test?

Reply:
It's been moved to `test_learn_rounding_digits`, I didn't delete it.